src/main/java/com/example/macro/user/exception/DuplicateUser.java

```diff
@@ -0,0 +1,7 @@
+package com.example.macro.user.exception;
+
+public class DuplicateUser extends RuntimeException {
```

Comment:
DuplicateUser and UserNotFoundException don't follow the same convention. Specify DuplicateUserException to match Exception naming convention.